tests/cypress/e2e/unit_tests/p1_fleet.spec.ts

```diff
@@ -453,3 +453,48 @@ if (/\/2\.9/.test(Cypress.env('rancher_version'))) {
       }
     )
   });
+
+describe('Test application deployment based on clusterGroup', { tags: '@p1'}, () => {
+  qase(25,
+    it("Fleet-25: Test install single application to the all available clusters using 'clusterGroup'", { tags: '@fleet-25' }, () => {
+      const repoName = 'default-single-app-cluster-group-25'
+      const key = 'key_env'
+      const value = 'value_prod'
+      const clusterGroupName = 'cluster-group-env-prod'
+      const dsClusterList = ["imported-0", "imported-1"]
+      const bannerMessageToAssert = 'Matches 2 of 3 existing clusters, including "imported-1"'
+
+      cy.accesMenuSelection('Continuous Delivery', 'Clusters');
+      cy.contains('.title', 'Clusters').should('be.visible');
+
+      // Assign label to the clusters 
+      dsClusterList.forEach(
+        (dsClusterName) => {
+          cy.assignClusterLabel(dsClusterName, key, value);
+        }
+      )
+
+      // Create group of cluster consists of same label.
+      // dsClusterList[1]: this will provide index 1 element of the array.
```

Comment:
Sorry, not sure what info brings this.

Reply:
Oops, that was a leftover, I will remove it in next patch set.